test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/IncludeSearchTests.cs

```diff
@@ -1352,6 +1352,21 @@ public async Task GivenARevIncludeIterateSearchExpressionWithCircularReference_W
             };
         }
 
+        [Fact]
+        [HttpIntegrationFixtureArgumentSets(DataStore.SqlServer)]
+        public async Task GivenARevIncludeIterateSearchExpressionWithInvalidTargetResourceType_WhenSearched_ShouldThrowResourceNotSupportedException()
+        {
+            string query = $"_include=Observation:subject:NotAResourceType";
```

Comment:
Should we add a test for _include=Observation:subject:

Reply:
Added test cases for empty/white space